create-lcp-tboot-policy_el6.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# This script will create the Launch Control Policy (lcp) and tboot policy
+# for a Measured Launch Environment (mle) and write the policy to the NVRAM on
+# the Trusted Platform Module (tpm) on the mobo.
+#
+# For complete details, and a cure for insomnia, read the complete documents
+# policy_v2.txt and lcptools2.txt found in /usr/share/doc/tboot-1.7.0/.
+
+
+if [ $UID -ne 0 ]; then
+    echo "This can only be executed as root.  Aborting."
+    exit 1
+fi
+
+if [ $# -ne 1 ]; then 
+    echo "Usage: $0 <20_character_passwrd>"
+    exit 1
+fi
+
+# TPM password - MUST BE 20 CHARACTERS!
+#PASSWORD="20 character passwrd"
+PASSWORD="$1"
+
+if [ `echo -n $PASSWORD | wc -c` -ne 20 ]; then
+    echo "Password is not 20 characters long.  Please try again."
+    exit 1
+fi
+
+# Clean up the files after we're done?
+CLEAN_UP=false
+
+if [[ ! -d /root/txt ]]; then
+  mkdir /root/txt
+fi
+cd /root/txt/
+
+# Clean up the files before we start!  vl.pol is simply appended to, not 
+# written over - might as well clean up everything else, too.
+rm -f mle_hash mle.elt pcrs pconf.elt list_unsig.lst privkey.pem pubkey.pem list_sig.lst list.pol list.data vl.pol
+
+if [ ! -e /boot/tboot.gz ]; then 
+    echo "/boot/tboot.gz does not exist - did you install tboot?"
+    echo "(yum -y install tboot)"
+    exit 1
+fi
+
+# a. Create hash for tboot.gz and store it in the mle_hash file:
+lcp_mlehash -c "logging=vga,serial,memory vga_delay=10" /boot/tboot.gz > mle_hash
```

Comment:
We should probably make the tboot params into an argument of the script for easy puppeting